W-20508532 eol tokenization gr (#968)
mulesoft/docs-gateway: mule-gateway/modules/ROOT/pages/policies-included-detokenization.adoc

:keywords: detokenization, api gateway, mule gateway, policy
:page-aliases: api-manager::policy-mule4-detokenization.adoc, policies::policies-included-detokenization.adoc

> [NOTE]

Reviewer: You can use the [banner](https://confluence.internal.salesforce.com/spaces/MTDT/pages/550316958/Managing+EOL+Content#ManagingEOLContent-WhenaProduct,Version,orFeatureReachesEOLorEOLSupport:~:text=has%20reached%20EOL-,Banner%20attribute%20at%20the%20top%20of%20the%20.adoc%20(below%20H1%20and%20above%20other%20attributes)%3A,-Turn%20on%20wrap) for this, since it's a full topic.

| Format
| Type of information that will be detokenized. Formats may vary depending on the selected service. Choose one format from the list.

> If you do not see any formats in the list, verify that the tokenization service selected contains at least one format.

Reviewer: ```suggestion
If you don't see any formats in the list, verify that the tokenization service selected contains at least one format.
```
